test/onnx/exporter/test_api.py

```diff
@@ -181,6 +193,47 @@ def forward(self, x):
         assert onnx_program is not None
         onnx_testing.assert_onnx_program(onnx_program)
 
+    def test_dynamic_shapes_supports_nested_input_model_with_input_names_assigned(self):
+        # kwargs can still be renamed as long as it's in order
+        input_names = ["input_x", "input_y", "input_z", "d", "e", "f"]
+
+        dynamic_axes = {
```

Comment:
Possible to add a test when a particular input is not specified here?

Reply:
I added f to the model output.

Reply:
Done